classes/NetCode.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * NetCode.js
+ */
+
+// Import JSON declaration of opcodes + types
+const opcodes = require("./../opcodes.json")
```

Comment:
These are not opcodes though. I'd say they're "operations", "ops", "commands", "cmd", whatever
And the same goes for the filename